debian/control

```diff
@@ -7,5 +7,5 @@ Standards-Version: 3.9.6.1
 
 Package: torwatchdog
 Architecture: all
-Depends: confighelper (>=0.3),gpgmailer (>=0.7),python-stem,python-socksipy,tor
+Depends: adduser (>= 3.11),confighelper (>=0.3),gpgmailer (>=0.7),python-daemon,python-stem,python-socks,tor
```

Comment:
python-socks does not exist in the Ubuntu 14.04 repositories, and python-socksipy is not in the Ubuntu 16.04 repositories. Debhelper supports alternative package names, though.